rosidl_typesupport_introspection_c/include/rosidl_typesupport_introspection_c/identifier.h

```diff
@@ -26,6 +26,10 @@ extern "C"
 ROSIDL_TYPESUPPORT_INTROSPECTION_C_PUBLIC
 extern const char * rosidl_typesupport_introspection_c__identifier;
 
+/// String identifying the typesupport introspection implementation in use.
+ROSIDL_TYPESUPPORT_INTROSPECTION_C_PUBLIC
+extern const char * rosidl_typesupport_introspection_c__identifier_v2;
```

Comment:
I would do this the other way around, so any code currently using `rosidl_typesupport_introspection_c__identifier` does not need to be changed.

```suggestion
ROSIDL_TYPESUPPORT_INTROSPECTION_C_PUBLIC
extern const char * rosidl_typesupport_introspection_c__identifier;

/// String identifying a previous version of the typesupport introspection implementation.
ROSIDL_TYPESUPPORT_INTROSPECTION_C_PUBLIC
extern const char * rosidl_typesupport_introspection_c__identifier_v1;
```